src/components/ProjectTagElem.tsx

```diff
@@ -0,0 +1,32 @@
+import { CloseOutlined } from '@ant-design/icons'
+import { ProjectTag } from 'models/project-tags'
+import { twMerge } from 'tailwind-merge'
+
+export default function ProjectTagElem({
+  tag,
+  className,
+  selected,
+  onClick,
+}: {
+  tag: ProjectTag
+  className?: string
+  selected?: boolean
+  onClick?: (tag: ProjectTag) => void
+}) {
+  return (
+    <div
+      onClick={() => onClick?.(tag)}
+      className={twMerge(
+        `cursor-pointer rounded-full py-1 px-3 ${
+          selected
+            ? 'bg-smoke-300 font-medium dark:bg-slate-400'
+            : 'bg-smoke-100 dark:bg-slate-600'
+        }`,
+        className,
+      )}
+    >
+      {selected ? <CloseOutlined className="mr-2" /> : null}
+      {tag.toUpperCase()}
```

Comment:
**suggestion** If we want to uppercase, we can use css to do so instead via `uppercase` tailwind class.

Wondering what you think about capitalizing instead of uppercasing 🤔 . Not sure

Reply:
ohhh down to use CSS here. went back and forth on all caps. generally I like all caps to visually differentiate "label" text from text that should be _read_.  the uppercase feels a bit better to me I think but I could go either way